scripts/load_mock_unit_data.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+# TODO: For PR testing convenience - revert when PR in sds-schema-definitions repo is merged
+# PR CONVENIENCE BLOCK START
+mkdir temp_sds_schema
+git clone -b update-example-file-names-with-test-identifier https://github.com/ONSdigital/sds-schema-definitions temp_sds_schema
+
+for path in temp_sds_schema/examples/*; do
+  folder=$(basename "$path")
+  rm -rf ./mock_data/"${folder}"
+  mv -f "${path}" ./mock_data/
+done
+
+rm -rf temp_sds_schema
+# PR CONVENIENCE BLOCK END
+
+
+# TODO: Below lines to be reverted when update-example-file-names-with-test-identifier branch is merged in sds-schema-definitions repo
```

Comment:
Difficult to test this part until the other PR merges as you mention, although the above seems to work and I am able to launch SDS schemas